glean-core/src/lib.rs

```diff
@@ -557,6 +563,15 @@ fn initialize_inner(
             Err(err) => log::error!("Unable to flush the preinit queue: {}", err),
         }
 
+        if !dispatcher::global::is_test_mode() {
```

Comment:
Why disabling it in `test_mode`?
(I'm not even sure we reliably set that in e.g. m-c)

Reply:
So, now we _more reliably_ set test_mode, at least in glean-core and the rlb :)